.github/workflows/v3_hatchet-worker-general-qa.yml

```
@@ -0,0 +1,60 @@
+  IMAGE_NAME: ${{ github.repository }}/hatchet-worker-general
+
+jobs:
+  build:
+    if: github.event.pull_request.draft == false
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)